### PR TITLE
RELATED: BB-2443 /gdc/app/account/bootstrap response extension

### PR DESCRIPTION
**Pull request by xMort**
There are two new optional pieces of information in /gdc/app/account/bootstrap response: clientId and dataProductId.

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/389/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1970/ : SUCCESS in 11m 29s

**Comment by lupko** [comment]
@xMort why is this hanging here? seems mergable to me. shall we? or shall we close this? :)

**Comment by xMort** [comment]
> @xMort why is this hanging here? seems mergable to me. shall we? or shall we close this? :) 

I'm waiting on final approval of webapp part (in the case it would have to be renamed or moved in the structure). When it is done, I will ask you to merge it.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/266/ : SUCCESS in 17s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/522/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2024/ : SUCCESS in 9m 42s

**Pull request merged**
